docs/modules/3d-tiles/api-reference/cesium-ion-loader.md

```diff
@@ -62,8 +62,31 @@ For i3dm and b3dm tiles:
 | ------------------------- | ------- | ------- | ------------------------------------- |
 | `['cesium-ion'].loadGLTF` | Boolean | `true`  | Fetch and parse any linked glTF files |
 
-If `options['cesium-ion'].loadGLTF` is `true`, GLTF loading can be controlled by providing [`GLTFLoader` options](modules/gltf/docs/api-reference/gltf-loader) via the `options.gltf` sub options.
+If `options['cesium-ion'].loadGLTF` is `true`, GLTF loading can be controlled by providing [`GLTFLoader` options](/docs/modules/gltf/api-reference/gltf-loader) via the `options.gltf` sub options.
 
 ## Data formats
 
 The same as `Tiles3DLoader`.
+
+### Tile Object
```

Comment:
Why are we adding this here? I don't think we should duplicate information - it will be very hard to keep up to date, we should link to the documentation in the `tiles3d` module...